optional_plugins/varianter_yaml_to_mux/avocado_varianter_yaml_to_mux/__init__.py

```diff
@@ -492,6 +493,13 @@ class YamlToMux(mux.MuxPlugin, Varianter):
     description = 'Multiplexer plugin to parse yaml files to params'
 
     def initialize(self, config):
+        """
+        :warning message: We are using `ast.literal_eval()` despite the
```

Comment:
the docstring is not following pep-257.

Reply:
@beraldoleal Done, and raised a v2 https://github.com/avocado-framework/avocado/pull/4002 since there are merge conflicts due to latest commits.